error.go

```diff
@@ -0,0 +1,5 @@
+package tprstorage
+
+import "github.com/giantswarm/microstorage"
+
+var invalidConfigError = microstorage.InvalidConfigError
```

Comment:
@xh3b4sd I made a PR which address it https://github.com/giantswarm/microerror/pull/4 let's discuss there if that makes sense.